saleor/graphql/product/mutations/products.py

```diff
@@ -802,6 +802,18 @@ def clean_attributes(
     @classmethod
     def clean_input(cls, info, instance, data):
         cleaned_input = super().clean_input(info, instance, data)
+
+        weight = cleaned_input.get("weight")
+        if weight and weight.value < 0:
```

Comment:
btw. can something have 0? if not that would have to be `<=`

Reply:
We talk about it yesterday, and in some cases, you want to set products (e.g. Post Cards) Weight to 0.

Reply:
Well, post cards have some not-null weight but I guess there might be someone that sells virtual goods among physical products so that makes sense anyway.